components/form/fields/RatingField.tsx

```diff
@@ -0,0 +1,23 @@
+import { Rating as MuiRatingField, Typography } from '@mui/material'
+
+type RatingFieldProps = {
+    name: string
+    field: any
```

Comment:
`field: { value: number, error: string}`

Reply:
Make this `type` reusable. Import it from utils/types, put it in the appropriate (create if needed) folder and import it here.